src/Symfony/Component/Translation/Bridge/PoEditor/composer.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "symfony/poeditor-translation-bridge",
+    "type": "symfony-bridge",
+    "description": "Symfony PoEditor Translation Bridge",
+    "keywords": ["poeditor", "translation", "provider"],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Mathieu Santostefano",
+            "homepage": "https://github.com/welcomattic"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=7.2.5",
+        "symfony/http-client": "^5.3",
```

Comment:
could work with `"symfony/http-client-contracts": "^2.4"` now, isn't it? (same in other providers)